autogen/agentchat/conversable_agent.py

```diff
@@ -1581,7 +1581,7 @@ async def a_generate_function_call_reply(
         if messages is None:
             messages = self._oai_messages[sender]
         message = messages[-1]
-        if "function_call" in message:
+        if "function_call" in message and message["function_call"]:
             func_call = message["function_call"]
```

Comment:
```suggestion
        func_call = message.get("function_call")
        if func_call:
```

Reply:
@artero with this change the PR is merge-able

Reply:
Hi @sonichi, I was on vacation and the invitation sent to me has expired. Could you please resend it?

Reply:
@ekzhu , could you help here? Thanks.

Reply:
I incorporated the suggestion and rebased my branch with the current main branch.​